shiny/playwright/controller/_expect.py

```diff
@@ -218,9 +221,13 @@ def expect_locator_values_in_list(
     # and all elements all unique, then it should have a count of `len(arr)`
     loc_inputs = loc_container.locator(loc_item)
     try:
-        playwright_expect(loc_inputs).to_have_count(len(arr), timeout=timeout)
+        if kwargs.get("alt_verify"):
+            sleep(1)  # to make up for not using to_have_count
+            assert loc_inputs.count() == len(arr)
```

Comment:
Would this work if we tried to check for any text values? Thinking that `.to_have_text()` works on an array of elements. Maybe it is jus the `.to_have_count()` _count_ part that is bad. 

```suggestion
            anything = re.compile(r".*")
            any_text_values = [anything for _ in range(len(arr))]
            playwright_expect(loc_inputs).to_have_text(any_text_values, timeout = timeout)
```

Reply:
unfortunately not. `.to_have_text()` is yielding the same results as the `.to_have_count()`

Reply:
Damn